src/build/build.ts

```diff
@@ -34,7 +34,7 @@ const buildDirectory = 'build/';
 
 export interface BuildOptions {
   swPrecacheConfig?: string;
-  insertDependencyLinks?: boolean;
+  prefetchDepedencies?: boolean;
```

Comment:
"insert" seemed meaningful here, in that the effect of this flag is to insert prefetch link tags, the effect isn't to do prefetching.